e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -79,7 +79,10 @@ describe("database prompt banner", () => {
 
     describe("full-app embeddings", () => {
       it("should render database prompt banner when logged in as an admin, an instance is on a paid plan, only have a single sample dataset, and is not white labeling", () => {
-        visitUrl({ url: "/", qs: { side_nav: false, logo: false } });
+        visitFullAppEmbeddingUrl({
```

Comment:
Modify the function that to have the same name as the one added in `e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js` to make it clearer and consistent.

The original function is defined in `e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js`, so it's acceptable to be named `visitUrl` in that file, but not in other files.